src/libraries/System.Private.Xml/src/System/Xml/Core/XmlWellFormedWriterAsync.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Are you adding the #nullable directive on top of the files and not to the csproj because the PRs don't cover the whole project files, right?

I assume once everything is done, we will remove this from each file and just enable it at the csproj level, right?

Reply:
correct x2

Reply:
Yep, since the project has too many files we add it individually until annotate all files